### ISSUE TITLE
Add keyboard navigation to page flipper

### ISSUE THREAD

**Issue by raae** [olavea/gatsby-theme-jam-trixie-hoods-and-unicorn-lillis-revenge-on-the-pirate-princess#14]
Let the reader flip pages with the keyboard.

**Comment by olavea** [comment]
I did some searches:
"keyboard navigation react" and keyboard navigation  
both on google and on https://stackoverflow.com/search?q=keyboard+navigation+gatsby

no joy

Where should I look?

**Comment by olavea** [comment]
But. I already HAVE keyboard navigation I can TAB to all I need.

**Comment by olavea** [comment]
So keyboard navigation means use arrow keys to flip pages. I re-watch the "Handling Events" video in Wes Bos's React for beginners course

**Comment by raae** [comment]
Yes, I should have been clearer.
